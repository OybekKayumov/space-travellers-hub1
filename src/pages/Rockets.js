import React from 'react';

function Rockets() {
  return (
    <div>
      Rockets
    </div>
  );
}

export default Rockets;
