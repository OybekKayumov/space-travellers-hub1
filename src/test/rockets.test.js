import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import elementStore from '../redux/configureStore';
import Rockets from '../pages/Rockets';

// arrangement
const rockets = [
  {
    desc: 'The SpaceX Rockets are good module for render',
    key: 'frame',
    id: 'rocky',
    type: 'module',
    name: 'rocky',
    image: 'url',
    reservation: false,
  },
];

describe('Test render the Rockets Component', () => {
  it('rockets section renders a hardcoded obj, passed as a prop', () => {
    // act
    const { container } = render(
      <Provider store={elementStore}>
        <Rockets key={rockets[0].id} rocket={rockets[0]} />
      </Provider>,
    );

    // assert
    expect(container).toMatchSnapshot();
  });
});
