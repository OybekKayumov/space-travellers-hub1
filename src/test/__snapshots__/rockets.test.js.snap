// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test render the Rockets Component rockets section renders a hardcoded obj, passed as a prop 1`] = `
<div>
  <div
    class="container"
  >
    <ul
      class="flex1"
    />
  </div>
</div>
`;
